README.md

```diff
@@ -19,6 +17,58 @@ features:
 For more details, please see the
 [documentation](https://kubernetes-csi.github.io/docs/Setup.html#csidriver-custom-resource-alpha).
 
+## Compatibility
+
+| Latest stable release                                                                                       | Branch                                                                                  | Compatible with CSI Version                                                                | Container Image                                 | Min K8s Version | Max K8s Version |
+| ----------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ | ----------------------------------------------- | --------------- | --------------- |
+| [cluster-driver-registrar v1.0.1](https://github.com/kubernetes-csi/cluster-driver-registrar/releases/tag/v1.0.1) | [release-1.0](https://github.com/kubernetes-csi/cluster-driver-registrar/tree/release-1.0) | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1 | 1.13            | -               |
+
+## Usage
+
+A UNIX domain socket is used by the cluster-driver-registrar:
+
+* CSI driver socket:
+  * Used by kubelet to interact with the CSI driver.
+  * Created by the CSI driver.
+  * Exposed on a Kubernetes node via hostPath somewhere other than the Kubelet.
```

Comment:
For the controller portion, the socket does not need to be exposed on the node.  Only an emptydir shared between driver and sidecar is sufficient